extensions/interactions/MusicNotesInput/directives/music-notes-input-validation.service.spec.ts

```diff
@@ -28,11 +28,19 @@ import {
   OutcomeObjectFactory,
 } from 'domain/exploration/OutcomeObjectFactory';
 
+import {AppConstants} from 'app.constants';
+import {Rule} from 'domain/exploration/rule.model';
+import {MusicNotesInputCustomizationArgs} from 'extensions/interactions/customization-args-defs';
+import cloneDeep from 'lodash/cloneDeep';
+
 describe('MusicNotesInputValidationService', () => {
   let validatorService: MusicNotesInputValidationService;
+  let customizationArgs: MusicNotesInputCustomizationArgs;
 
   let currentState: string;
-  let goodAnswerGroups: AnswerGroup[], goodDefaultOutcome: Outcome;
+  let answerGroups: AnswerGroup[],
+    goodAnswerGroups: AnswerGroup[],
+    goodDefaultOutcome: Outcome;
```

Comment:
Is this the correct formatting? Maybe we can add them in separate lines, what do you think?

Reply:
Looked at a couple of the others "-input-validation-service.spec.ts" when I created the tests and saw that in some files there are "goodAnswerGroups" and in some there are "answerGroups"

Some examples are:
https://github.com/oppia/oppia/blob/develop/extensions/interactions/ItemSelectionInput/directives/item-selection-input-validation.service.spec.ts
That uses "goodAnswerGroups" at row 41.

and
https://github.com/oppia/oppia/blob/develop/extensions/interactions/GraphInput/directives/graph-input-validation.service.spec.ts
That I used the most to understand how to write the tests. Here they use "answerGroups" at row 43.

There are more examples but it seems like there is either "answerGroups" or "goodAnswerGroups" and they are all in the same row as "goodDefaultOutcome" that always seem to be included. I cannot find any instance of both "answerGroups" and "goodAnswerGroups" being included together.

Should I move out the "answerGroups" to a seperate line or keep it as it is?

Thanks for the quick reply!

Reply:
Thanks for the explanation, looks good.